### PR TITLE
Update gimp - binary

### PR DESCRIPTION
**Pull request by fepegar**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] `sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not already refused in [closed issues].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/gimp.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@
   homepage 'https://www.gimp.org/'
 
   app 'GIMP.app'
+  binary "#{appdir}/GIMP.app/Contents/MacOS/GIMP", target: 'gimp'
```
Comment (commitay) [inline Casks/gimp.rb]: @fepegar Have you tested this? It doesn't seem to work.

```
$ gimp
Setting up environment...
Enabling internal Python...
Launching GIMP...
/usr/local/bin/gimp: line 99: /usr/Contents/MacOS/gimp-bin: No such file or directory
/usr/local/bin/gimp: line 99: exec: /usr/Contents/MacOS/gimp-bin: cannot execute: No such file or directory
```

**Comment by fepegar** [comment]
I went through the steps of the issue and it went ok. I did forget to try
the command after. Maybe a second binary is needed:
```

binary "#{appdir}/GIMP.app/Contents/MacOS/GIMP-bin", target: 'gimp-bin'

```


I will modify it soon and commit again after trying properly.


El 7 sept. 2017 2:14 a. m., "commitay" <notifications@github.com> escribió:

*@commitay* commented on this pull request.
------------------------------

In Casks/gimp.rb
<https://github.com/caskroom/homebrew-cask/pull/38338#discussion_r137427496>
:

> @@ -7,6 +7,7 @@
   homepage 'https://www.gimp.org/'

   app 'GIMP.app'
+  binary "#{appdir}/GIMP.app/Contents/MacOS/GIMP", target: 'gimp'

@fepegar <https://github.com/fepegar> Have you tested this? It doesn't seem
to work.

$ gimp
Setting up environment...
Enabling internal Python...
Launching GIMP...
/usr/local/bin/gimp: line 99: /usr/Contents/MacOS/gimp-bin: No such
file or directory
/usr/local/bin/gimp: line 99: exec: /usr/Contents/MacOS/gimp-bin:
cannot execute: No such file or directory

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/caskroom/homebrew-cask/pull/38338#pullrequestreview-61092452>,
or mute the thread

**Comment by commitay** [comment]
Also, this won't work unless the GIMP app has completed its first launch setup.

**Comment by commitay** [comment]
Thanks for the PR @fepegar, going to pass on this because it doesn't work unless the GUI app is launched first.

Also, I can't find anything in the upstream docs about symlinking the `app` binaries. We generally prefer that upstream recommends creating binaries.

**Comment by fepegar** [comment]
Well I tried. I hope someone figures this out one day. Thanks for the reviews @commitay.

**Pull request closed without merging**